### PR TITLE
first test

### PR DESCRIPTION
**Pull request by shaneknapp**
## What changes were proposed in this pull request?

¯\\\_(ツ)\_/¯

## How was this patch tested?

jenkins!

Please review http://spark.apache.org/contributing.html before opening a pull request.

### CODE REVIEW

**Comment by shaneknapp** [comment]
test this please

**Comment by shaneknapp** [comment]
test this please

**Comment by shaneknapp** [comment]
test this please

**Comment by shaneknapp** [comment]
test this please

**Comment by shaneknapp** [comment]
test this please

**Comment by shaneknapp** [comment]
test this please

**Comment by shaneknapp** [comment]
test this please

**Comment by HyukjinKwon** [comment]
Hey @shaneknapp! I happened to come here while exploring Jenkins builds (while having a lunch). The error:

```
17-12-18 17:46:13.779 - stdout> Traceback (most recent call last):
2017-12-18 17:46:13.779 - stdout>   File "/home/jenkins/workspace/ubuntuSparkPRB/target/tmp/test9115819157222735197.py", line 2, in <module>
2017-12-18 17:46:13.779 - stdout>     from pyspark.sql import SparkSession
2017-12-18 17:46:13.779 - stdout>   File "<frozen importlib._bootstrap>", line 971, in _find_and_load
2017-12-18 17:46:13.779 - stdout>   File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
2017-12-18 17:46:13.779 - stdout>   File "<frozen importlib._bootstrap>", line 656, in _load_unlocked
2017-12-18 17:46:13.78 - stdout>   File "<frozen importlib._bootstrap>", line 626, in _load_backward_compatible
2017-12-18 17:46:13.78 - stdout>   File "/tmp/test-spark/spark-2.0.2/python/lib/pyspark.zip/pyspark/__init__.py", line 44, in <module>
2017-12-18 17:46:13.78 - stdout>   File "<frozen importlib._bootstrap>", line 971, in _find_and_load
2017-12-18 17:46:13.78 - stdout>   File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
2017-12-18 17:46:13.78 - stdout>   File "<frozen importlib._bootstrap>", line 656, in _load_unlocked
2017-12-18 17:46:13.78 - stdout>   File "<frozen importlib._bootstrap>", line 626, in _load_backward_compatible
2017-12-18 17:46:13.78 - stdout>   File "/tmp/test-spark/spark-2.0.2/python/lib/pyspark.zip/pyspark/context.py", line 33, in <module>
2017-12-18 17:46:13.78 - stdout>   File "<frozen importlib._bootstrap>", line 971, in _find_and_load
2017-12-18 17:46:13.78 - stdout>   File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
2017-12-18 17:46:13.78 - stdout>   File "<frozen importlib._bootstrap>", line 656, in _load_unlocked
2017-12-18 17:46:13.78 - stdout>   File "<frozen importlib._bootstrap>", line 626, in _load_backward_compatible
2017-12-18 17:46:13.78 - stdout>   File "/tmp/test-spark/spark-2.0.2/python/lib/pyspark.zip/pyspark/java_gateway.py", line 25, in <module>
2017-12-18 17:46:13.78 - stdout>   File "/home/anaconda/envs/py3k/lib/python3.6/platform.py", line 886, in <module>
2017-12-18 17:46:13.78 - stdout>     "system node release version machine processor")
2017-12-18 17:46:13.78 - stdout>   File "/tmp/test-spark/spark-2.0.2/python/lib/pyspark.zip/pyspark/serializers.py", line 381, in namedtuple
2017-12-18 17:46:13.78 - stdout> TypeError: namedtuple() missing 3 required keyword-only arguments: 'verbose', 'rename', and 'module'
```

Seems related with https://issues.apache.org/jira/browse/SPARK-19019 and  it was caused by Spark-2.0.2 and Python 3.6.0 combination ...

Just wanted to help just in cases :sweat_smile:!

**Comment by shaneknapp** [comment]
ah, interesting.  thanks for the heads up!  :)

(as you can see here i'm hoping to get our builds off of centos6 asap and
on to ubuntu)

**Comment by shaneknapp** [comment]
alrighty then...  i've set up the ubuntu py3k environment to be a clone of the centos one (python 3.4.5, all conda and pip paks, etc).

test this please

**Comment by shaneknapp** [comment]
test this please

**Comment by shaneknapp** [comment]
test this please

**Comment by shaneknapp** [comment]
test this please

**Comment by shaneknapp** [comment]
test this please

File: dev/run-tests.py [hunk 1]
```diff
@@ -576,8 +576,8 @@ def main():
                                 for f in changed_files):
         # run_java_style_checks()
         pass
-    if not changed_files or any(f.endswith(".py") for f in changed_files):
-        run_python_style_checks()
+    # if not changed_files or any(f.endswith(".py") for f in changed_files):
+        # run_python_style_checks()
```
Comment (shaneknapp) [inline dev/run-tests.py]: i'll revert this as it was a cert issue on the machine...  i'm still unsure why the cert issue actually happened, but regardless, it's fixed.

**Comment by shaneknapp** [comment]
test this please

**Pull request closed without merging**